core/tests/karma-globals.js

```diff
@@ -1,13 +1,13 @@
 // Copyright 2016 The Oppia Authors. All Rights Reserved.
 //
-// Licensed under the Apache License, Version 2.0 (the "License");
+// Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
@seanlip Ah yeah, it happened because I did a search replace for the double quote linting rule. I'll change it back.